spike_main/spike.cc

```diff
@@ -64,6 +64,7 @@ static void help(int exit_code = 1)
   fprintf(stderr, "  --dump-dts            Print device tree string and exit\n");
   fprintf(stderr, "  --dtb=<path>          Use specified device tree blob [default: auto-generate]\n");
   fprintf(stderr, "  --disable-dtb         Don't write the device tree blob into memory\n");
+  fprintf(stderr, "  --disable-ns16550     Disable default ns16550 UART device\n");
```

Comment:
I'm not sure this should be called "UART device" (maybe just serial or console device)?